.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+<!--
+Have you read Pulsar's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/pulsar-edit/.github/blob/master/CODE_OF_CONDUCT.md
+Do you want to ask a question? Are you looking for support? The Pulsar Discussion tab is the best place for getting support: https://github.com/orgs/pulsar-edit/discussions
+---
+Keep in mind that Pulsar is highly customizable in a number of ways and we strongly prefer that you consider these options before filing this issue:
+* https://flight-manual.atom.io/using-atom/sections/basic-customization/: tweak Pulsar's configuration, styles, and keybindings.
+* https://flight-manual.atom.io/using-atom/sections/atom-packages/: install a community package.
+* https://flight-manual.atom.io/hacking-atom/: use the Pulsar API in your init script, to create a package, or to enhance an existing package.
+If you're convinced that none of these options are appropriate for the feature you want, please explain why that's the case by completely filling out the issue template below.
+Also note that the Pulsar team has finite resources so it's unlikely that we'll work on feature requests. If we're interested in a particular feature however, we'll follow up and ask you to submit an RFC to talk about it in more detail.
+-->
+
+## Summary
+
+<!-- One paragraph explanation of the feature. -->
+
+## Motivation
+
+<!-- Why are we doing this? What use cases does it support? What is the expected outcome? -->
+
+## Are there any other possible alternatives? (Is this really the best solution or only option to add this feature? Or could it be a plugin?)
```

Comment:
```Suggestion
### Are there any other possible alternatives? (Is this really the best solution or only option to add this feature? Could it be a plugin?)
```
Is this intended to be an H2 sizing? I'd assume not, so it can probably be downgraded to an H3

Otherwise, if it is, feel free to disregard.

Reply:
I've gone ahead and changed the grammar you suggested. Otherwise I believe this is intended to be used as H2 sizing, since all other points in that are such.